docs/content/guides/operator/snapshots.mdx

```diff
@@ -54,9 +88,49 @@ When you restore a Full node from a snapshot, write it to the path `/opt/sui/db/
 
 :::
 
-## S3 buckets used per environment
+### Restoring using Formal snapshots
+
+To restore using a Formal snapshot, you can use the `sui-tool` binary. `sui-tool` can be downloaded along with other sui binaries.
+See [Install Sui](./getting-started/sui-install.mdx) for more details.
+
+The following steps can be used to restore a node from a formal snapshot:
+
+1. If it's running, stop the node.
+2. Run the command:
+   ```
+   sui-tool download-db-snapshot --epoch "<EPOCH-NUMBER>" --genesis "<PATH-TO-GENESIS-BLOB>" \
+        --formal --network "<NETWORK>" --path <PATH-TO-NODE-DB> --num-parallel-downloads 50 --no-sign-request --snapshot-bucket-type=s3
+   ```
```

Comment:
This command only showcases a way for them to use our hosted formal snapshots. But hopefully many will be storing and using their own. Could we start with showing the general case, and then after, show the special case (using ours), with a note that this would be using ours?

In the general case, we would omit `--network` as this allows `sui-tool` to generate sane defaults for ML buckets and such, and then we would add `--snapshot-bucket <bucket name>` and `--snapshot-bucket-type <type, e.g. gcs>`